### PR TITLE
Pallet scanning

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

**Comment by stdavis** [comment]
@steveoh If you have the time this afternoon for a review of this, I'd love to get this merged before tonight so that I can deploy the new version of locate tomorrow morning.

File: src/forklift/cli.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
 from git import Repo
-from importlib import import_module
+from imp import load_source
```
Comment (steveoh) [inline src/forklift/cli.py]: tell me about this...

Comment (stdavis) [inline src/forklift/cli.py]: This allow for loading a module using the full path to the file thus guaranteeing uniqueness in our instance.

**Comment by steveoh** [comment]
LGTM, just make sure you don't rebase in the old bad tests

**Comment by stdavis** [comment]
Merged.

**Pull request closed without merging**